lib/wallet/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs

```diff
@@ -3992,6 +4067,80 @@ prop_balanceTransactionExistingReturnCollateral
             e -> counterexample (show e) False
 
+prop_bootstrapWitnesses
+    :: (Word8 -> WriteTx.InAnyRecentEra Cardano.Tx -> Property)
+    -> Word8
+    -- ^ Number of bootstrap witnesses.
+    --
+    -- Testing with [0, 255] should be sufficient.
+    -> AnyRecentEra
+    -> Cardano.NetworkId
+    -- ^ Network - will be encoded inside the witness.
+    -> Index 'WholeDomain 'AccountK
+    -- ^ Account index - will be encoded inside the witness.
+    -> Index 'WholeDomain 'CredFromKeyK
+    -- ^ Index for the first of the 'n' addresses.
+    -> Property
+prop_bootstrapWitnesses
+    p n (AnyRecentEra (era :: RecentEra era)) net accIx addr0Ix =
+    let
+        -- Start incrementing the ixs upward, and if we reach 'maxBound', loop
+        -- around, to ensure we always have 'n' unique indicies.
+        addrIxs = take (fromIntegral n)
+                $ [addr0Ix .. maxBound] ++ [minBound .. addr0Ix]
+        body = emptyCardanoTxBody
+        wits :: [Cardano.KeyWitness era]
+        wits = map (dummyWitForIx body) addrIxs
+    in
+        p n (WriteTx.InAnyRecentEra era $ Cardano.Tx body wits)
+  where
+    emptyCardanoTxBody =
+        let
+            Cardano.Tx body _ = WriteTx.toCardanoTx @era $ WriteTx.emptyTx era
+        in body
+
+    mw = SomeMnemonic $ either (error . show) id
+        (entropyToMnemonic @12 <$> mkEntropy "0000000000000000")
+    rootK = Byron.generateKeyFromSeed mw mempty
+    pwd = mempty
+
+    dummyWitForIx
+        :: Cardano.TxBody era
+        -> Index 'WholeDomain 'CredFromKeyK
+        -> Cardano.KeyWitness era
+    dummyWitForIx body ix =
+        let
+            accK = Byron.deriveAccountPrivateKey pwd rootK accIx
+            addrKeyAtIx i = Byron.deriveAddressPrivateKey pwd accK i
+
+            addrK = addrKeyAtIx $ toEnum $ fromEnum ix
+            addr = case net of
+                Cardano.Mainnet ->
+                    paymentAddress @'Mainnet $ publicKey addrK
+                Cardano.Testnet _magic ->
+                    -- The choice of network magic here is not important. The
+                    -- size of the witness will not be affected by it. What may
+                    -- affect the size, is the 'Cardano.NetworkId' we pass to
+                    -- 'mkByronWitness' above.
+                    paymentAddress @('Testnet 0) $ publicKey addrK
+        in
+            case era of
+                RecentEraBabbage ->
+                    mkByronWitness body net addr (getRawKey addrK, pwd)
+                RecentEraAlonzo  ->
+                    mkByronWitness body net addr (getRawKey addrK, pwd)
+
+
+serializedSize :: forall era. Cardano.IsCardanoEra era => Cardano.Tx era -> Int
+serializedSize = BS.length
+    . serialisedTx
+    . sealedTxFromCardano
+    . Cardano.InAnyCardanoEra (Cardano.cardanoEra @era)
+
+(.>=) :: (Show a, Ord a) => a -> a -> Property
+a .>= b = counterexample (show a <> " < " <> show b)
+        $ property $ a >= b
```

Comment:
Suggestion 1: use `.>=.`, as it follows the existing QC style of defining new versions of existing operators with `.`.
Suggestion 2: how about putting this function in `Test.QuickCheck.Extra`?

```suggestion
(.>=.) :: (Show a, Ord a) => a -> a -> Property
a .>=. b = counterexample (show a <> " < " <> show b)
    $ property $ a >= b
```